gdal/ogr/ogrsf_frmts/ngw/drv_ngw.html

```diff
@@ -0,0 +1,217 @@
+
+<h2>Driver</h2>
+
+The driver available since GDAL/OGR >= <b>2.4</b>. This driver can connect to
+the services implementing the NextGIS Web API. GDAL/OGR must be built with Curl
+support in order for the NGW driver to be compiled.
+
+The driver supports read and write operations.
+
+<h2>Dataset name syntax</h2>
+
+The minimal syntax to open a NGW datasource is:
+
+NGW:[NextGIS Web URL][/resource/][resource identifier]
+
+<ul>
+    <li><b>NextGIS Web URL</b> may be an url to nextgis.com cloud service (for
+        example, https://demo.nextgis.com), or some other url including port and
+        additional path (for example, http://192.168.1.1:8000/test).
+    <li><b>resource</b> is mandatory keyword dividing resource identifier from
+        the rest of URL.
+    <li><b>resource identifier</b> this is positive number from 0 and above. This
+        may be an resource group, vector, PostGIS or raster layer, style.
+</ul>
+
+If identifier is resource group, all vector layers, PostGIS, raster layers, styles
+will listed as child resources. In other case this will be a separate layer.
+
+<h2>Configuration options</h2>
+
+The following configuration options are available:
+
+<ul>
+    <li><b>NGW_USERPWD</b>: User name and password separated with colon. Optional
+        and can be set using open options.
+    <li><b>NGW_BATCH_MODE</b>: If set to YES - all edits (insert, update) will
+        cache and executes if cache limit exceeded or sync to disk executes. Delete
+        operation will executes immediately.
+    <li><b>NGW_PAGE_SIZE</b>: If supported by server, fetch features from
+        remote server will use paging. The -1 value  disable paging even it
+        supported by server.
+</ul>
+
+<h2>Authentication</h2>
+
+Any operations (read, write, get metadata, chage properties, etc.) may require
+an authenticated access. Authenticated access is obtained by specifying user name
+and password in open, create or configuration options.
+
+<h2>Geometry</h2>
+
+NextGIS Web supports only one geometry column. Default spatial reference is Web
+Mercator (EPSG:3857). The following geometry types are available:
+
+<ul>
+    <li>POINT
+    <li>LINESTRING
+    <li>POLYGON
+    <li>MULTIPOINT
+    <li>MULTILINESTRING
+    <li>MULTIPOLYGON
+</ul>
+
+<h2>Field data types</h2>
+
+NextWeb supports only following field types:
+
+<ul>
+    <li>OFTInteger
+    <li>OFTInteger64
+    <li>OFTReal
+    <li>OFTString
+    <li>OFTDate
+    <li>OFTTime
+    <li>OFTDateTime
+</ul>
+
+<h2>Paging</h2>
+
+Features are retrieved from the server by chunks if server support this feature.
+This number can be altered with the NGW_PAGE_SIZE configuration option or
+PAGE_SIZE open option.
+
+<h2>Write support</h2>
+
+Datasource and layers creation and deletion is possible.
+Write support is only enabled when the datasource is opened in update mode and
+user have permissions.
+
+Vector and PostGIS layers insert and update operations may cached if BATCH_MODE
```

Comment:
may *be* cached

Reply:
Fixed per 4579aeab927d70ddab6142340bf5550c6789b42e